src/garage/tf/models/gru.py

```diff
@@ -28,10 +28,10 @@ def gru(name,
             hidden state is trainable.
 
     Return:
-        outputs (tf.Tensor): Entire time-series outputs.
-        output (tf.Tensor): Step output.
-        hidden (tf.Tensor): Step hidden state.
-        hidden_init_var (tf.Tensor): Initial hidden state.
+        tf.Tensor: Entire time-series outputs, with shape :math:`(N, T, S^*)`.
```

Comment:
you need to start this comment `r"""` so that it won't cause an error. this applies to any comment including `:math:` environment.